docs/reference/api.yaml

```diff
@@ -1695,6 +1695,138 @@ paths:
         - event
       security:
         - Bearer: []
+  /1.6/events/webhooks:
+    get:
+      operationId: WebHookList
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: WebHooks list.
+          schema:
+            type: array
+            items:
+              type: object
+              $ref: '#/definitions/WebHook'
+        '204':
+          description: No content.
+        '401':
+          description: Unauthorized.
+          schema:
+            $ref: '#/definitions/ErrorMessage'
+      tags:
+        - event
+      security:
+        - Bearer: []
+    post:
+      operationId: WebHookCreate
+      consumes:
+        - application/json
+      parameters:
+        - name: webhook
+          required: true
+          in: body
+          schema:
+            $ref: '#/definitions/WebHook'
+      responses:
+        '200':
+          description: WebHook created.
+        '400':
+          description: Invalid data
+          schema:
+            $ref: '#/definitions/ErrorMessage'
+        '401':
+          description: Unauthorized.
+          schema:
+            $ref: '#/definitions/ErrorMessage'
+        '409':
+          description: WebHook already exists.
+          schema:
+            $ref: '#/definitions/ErrorMessage'
+      tags:
+        - event
+      security:
+        - Bearer: []
+  /1.6/events/webhooks/{name}:
+    parameters:
+      - name: name
+        in: path
+        required: true
+        type: string
+        minLength: 1
+        description: WebHook name.
+    get:
+      operationId: WebHookInfo
```

Comment:
I think we agreed to use `Get` here, e.g: `WebHookGet`

Reply:
👍